pages/workspaces/[workspaceId]/contributors.tsx

```diff
@@ -40,7 +42,25 @@ interface WorkspaceContributorsPageProps {
 export default function WorkspaceContributorsPage({ workspace }: WorkspaceContributorsPageProps) {
   const router = useRouter();
   const range = router.query.range ? Number(router.query.range as string) : 30;
-  const { data, error: hasError } = useGetWorkspaceContributors({ workspaceId: workspace.id, range });
+  const [page, setPage] = useState(1);
```

Comment:
refactor (non-blocking): I don't think we do this in the app elsewhere yet (🤔|maybe?), but the page should be in the querystring as well.

I'll create an issue for adding the `page` to all the pages that use paging.